web-wallet/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Add dark theme [#1466]
+- Add dark mode support [#1466]
```

Comment:
I feel like "Dark mode" is more widespread as a term nowadays, also it matches the setting within the settings.